stl/inc/xmemory

```diff
@@ -73,27 +73,18 @@ _INLINE_VAR constexpr size_t _New_alignof = (_STD max)(alignof(_Ty),
 
 // STRUCT _Default_allocate_traits
 struct _Default_allocate_traits {
-    __declspec(allocator) static
-#ifdef __clang__ // TRANSITION, MSVC support for constexpr dynamic allocation
-        _CONSTEXPR20_DYNALLOC
-#endif // __clang__
-        void* _Allocate(const size_t _Bytes) {
+    __declspec(allocator) static _CONSTEXPR20_DYNALLOC void* _Allocate(const size_t _Bytes) {
         return ::operator new(_Bytes);
     }
 
 #ifdef __cpp_aligned_new
-    __declspec(allocator) static
-#ifdef __clang__ // TRANSITION, MSVC support for constexpr dynamic allocation
-        _CONSTEXPR20_DYNALLOC
-#endif // __clang__
+    __declspec(allocator) static _CONSTEXPR20_DYNALLOC

```

Comment:
Same here, this will break once `_CONSTEXPR20_DYNALLOC` turns into `constexpr` for MSVC